### ISSUE TITLE
Is this mod Dead or abandon

### ISSUE THREAD

**Issue by ModernGamingWorld** [PrinceOfAmber/Storage-Network#501]
I like many others have been waiting on fixes for the server-side crash for some time now on 1.19.2 and the Hosting Companies who run servers for our mod packs cant push to our new versions cause this mod is broken.

I did see that the mod developer was active 6 days ago updating some mod called FutureLibrary aka FLIB

**Comment by HalcyonSeas73** [comment]
If the mod developer is active and there's already an issue made just be patient, a fix will come when the author has time.

**Comment by HalcyonSeas73** [comment]
likely duplicate of #477

**Comment by Lothrazar** [comment]
sorry i didnt expect JEI and/or forge to break all my shit again lol
